.github/workflows/test.yml

```diff
@@ -12,15 +12,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix: # https://docs.djangoproject.com/en/stable/faq/install/#what-python-version-can-i-use-with-django
-        django-version: ["3.2", "4.2", "5.0", "5.1", "5.2rc1"]
+        django-version: ["4.2", "5.0", "5.1", "5.2rc1"]
```

Comment:
@auvipy Are you OK with dropping Django v3.2 support?

Would this fail if we had "5.2" instead of "5.2rc1"?

Reply:
For the moment yes.